org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/OverviewPage.java

```diff
@@ -349,6 +354,17 @@ private void createArtifactSection(FormToolkit toolkit, Composite composite, Wid
     artifactPackagingCombo.add("ear"); //$NON-NLS-1$
     artifactPackagingCombo.add("pom"); //$NON-NLS-1$
     artifactPackagingCombo.add("maven-plugin"); //$NON-NLS-1$
+
```

Comment:
Maybe it would be worth here to use a TreeSet<String> to first collect all packing types (initialized with the hard-coded defaults) so there is no need for filtering and no possible duplicates.